command-sctl.wrapper

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -eu
+
+sctlkey="$(snapctl get sctlkey)"
+if [ -z "$sctlkey" ]; then
+    echo "sctl is not configured. Please snap set sctl sctlkey=<YOUR_KMS_KEY_URI>"
+    exit 1
+fi
+
+# FIXME: Gross hack
```

Comment:
as someone out of the loop, perhaps include a brief description of what the hack is for?